.circleci/config.yml

```diff
@@ -6668,7 +6668,7 @@ workflows:
           build_environment: "pytorch-linux-xenial-cuda9.2-cudnn7-py3-gcc7-build"
           docker_image: "308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9.2-cudnn7-py3-gcc7"
       - pytorch_linux_test:
-          name: pytorch_linux_xenial_cuda9_2_cudnn7_py3_gcc7_test1
+          name: pytorch_linux_xenial_cuda9_2_cudnn7_py3_gcc7_test
```

Comment:
is cu92 significantly faster than cuda10/11?

Reply:
I'm not sure, but that particular test is not run in CI (not marked important in the CONFIG TREE)

Reply:
It is run in CI, but only on master (i.e. important tests run on both master and PRs, while others run only on master)